akvo/rsr/filters.py

```diff
@@ -98,26 +105,47 @@ def get_locations(location, locations):
 def location_choices(qs):
```

Comment:
I've tried to make the code a little more readable. Let me know if there's anything else that can be done to make it more so.